overview.js

```diff
@@ -446,18 +446,8 @@ var dtpOverview = Utils.defineClass({
                     return Clutter.EVENT_PROPAGATE;
                 }
 
-                let visibleView;
-                views.every(function(v, index) {
-                    if (v.view.actor.visible) {
-                        visibleView = index;
-                        return false;
-                    }
-                    else
-                        return true;
-                });
-
                 if(Me.settings.get_boolean('animate-show-apps')) {
-                    let view = views[visibleView].view;
+                    let view = views.find(v => v.view.actor.visible).view;
```

Comment:
Thanks l3nn4rt! Although we can't use `Array.prototype.find` as it isn't available in older gnome-shell versions :( 

For this , please use https://github.com/home-sweet-gnome/dash-to-panel/blob/8d20fb82c640b7fc4bc3edd2df015ec851c0cc39/utils.js#L324.

Reply:
Oh yes, I'll fix it right now! Thank you for reviewing :+1: